galsim/transform.py

```diff
@@ -20,7 +20,7 @@
 """
 
 import galsim
-import numpy
+import numpy as np
```

Comment:
I may be outvoted on this one, but I kind of hate this tendency of people to redefine modules with `import` as `some-two-character-abbreviation`.  `np` for `numpy` is pretty common, so it's not particularly unclear.  But in general I think it's worth the extra typing (only three characters in this case!) to be explicit about the real module name.